dev/cognitiveservices/data-plane/Language/analyzeconversations-authoring.json

```diff
@@ -2331,6 +2428,54 @@
       },
       "x-ms-client-name": "EntityEvaluationSummary"
     },
+    "ConversationalAnalysisAuthoringEvaluationJobResult": {
+      "required": [
+        "evaluationOptions",
+        "modelLabel",
+        "trainingConfigVersion",
+        "percentComplete"
+      ],
+      "type": "object",
+      "properties": {
+        "evaluationOptions": {
+          "description": "Represents the options used running the evaluation.",
+          "$ref": "#/definitions/ConversationalAnalysisAuthoringEvaluationOptions"
+        },
+        "modelLabel": {
+          "description": "Represents trained model label.",
+          "type": "string"
+        },
+        "trainingConfigVersion": {
+          "description": "Represents training config version.",
+          "type": "string"
+        },
+        "percentComplete": {
+          "format": "int32",
+          "description": "Represents progress percentage.",
+          "type": "integer"
```

Comment:
Generally, percentages are floats with a range of 0.0..1.0. It would be best to remain consistent.

Reply:
As discussed, we leave it to you but in the future, any dissimilar models we recommend floats for percentages e.g., `confidenceScore` and `confidenceScoreThreshold`.